testing_and_setup/compass/ocean/drying_slope/marsh_flooding/100m/config_init1.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<config case="init_step1">
+
+  <add_link source="../../../meshes/100m/build_100m_mesh/100mmesh.nc" dest="mesh.nc"/>
+  <add_link source="../../../meshes/100m/build_100m_mesh/graph.info" dest="graph.info"/>
+	<add_executable source="model" dest="ocean_model"/>
+
+	<namelist name="namelist.ocean" mode="init">
+		<template file="init_template.xml" path_base="script_configuration_dir"/>
+		<option name="config_use_variable_drag">.true.</option>
+		<option name="config_idealized_profile">.true.</option>
+		<option name="config_idealized_profile_Lcoast">0.3</option>
+		<option name="config_idealized_profile_Scoast">0.001</option>
+		<option name="config_idealized_profile_Lshore">0.6</option>
```

Comment:
Since this appears to be value that changed, recommend using templates so only this changed value shows up.  This was done for the WD cases for hurricanes, for example.